Create 2024-03-21.md diagnostic_type_code returns all values (#122)
department-of-veterans-affairs/developer-portal-content: content/verification/veteran-verification/release-notes/2024-03-21.md

> We've updated the response for /disability ratings. Diagnostic_type_code under individual_ratings will now include all values associated with the disability rating from the VA source system. Our response concatenates values when more than one is provided for a disability rating, inserting a hypen character "-" between each value.

Reviewer: ```suggestion
We've updated the response for /disability ratings. Diagnostic_type_code under individual_ratings will now include all values associated with the disability rating from the VA source system. Our response concatenates values when more than one is provided for a disability rating, inserting a hypen character "-" between each value.

To learn more, review the [Veteran Service History and Eligibility v2 documentation](https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current).
```

> We have updated the response for /disability ratings. Diagnostic_type_code under individual_ratings will now include all values associated with the disability rating from the VA source system. The lighthouse response concatenates values when more than one is provided for a disability rating, inserting a hypen character "-" between each value.

Reviewer: Long story, but we're trying to not refer to ourselves as "Lighthouse" to public party consumers. Just an fyi! So "our" will work here instead.

```suggestion
We've updated the response for /disability ratings. Diagnostic_type_code under individual_ratings will now include all values associated with the disability rating from the VA source system. Our response concatenates values when more than one is provided for a disability rating, inserting a hypen character "-" between each value.
```
